### ISSUE TITLE
missing the createBuilder method when installing through composer

### ISSUE THREAD

**Issue by shadowice222** [jordanbrauer/unit-converter#107]

**Comment by StenKoning** [comment]
+1

You can solve this temporary by adding the following to your require: "jordanbrauer/unit-converter": "master"

And adding to "repositories":

```
{
        "type": "package",
        "package": {
          "name": "jordanbrauer/unit-converter",
          "version": "master",
          "dist": {
            "type": "zip",
            "url": "https://github.com/jordanbrauer/unit-converter/archive/master.zip",
            "reference": "master"
          },
          "autoload": {
            "classmap": ["."]
          }
        }
      }
```

Then running composer update

**Comment by jordanbrauer** [comment]
Hey! That's is for the contribution. :)

I'll fix this ASAP with a fresh build and version bump, later this  evening the builder will be available in version `0.7.2`.

**Comment by jordanbrauer** [comment]
@shadowice222 @StenKoning 

This is now fixed in version `0.7.1`! To upgrade you can either manually change the version in your `composer.json` file, or you can update via the command line (probably the safest if you have other dependencies in your project as well).

#### Command Line Upgrade

Simply re-require the package with the new version! 

```bash
$ composer require jordanbrauer/unit-converter "^0.7.1"
```

#### Manual Composer Upgrade

Update the version for your require block.

```json
"require": {
    "jordanbrauer/unit-converter": "^0.7.1"
}
```

Then run an upgrade (might update other packages if you have them)

```bash
$ composer update
```
